.github/workflows/check-and-release-main.yaml

```diff
@@ -33,15 +41,22 @@ jobs:
     - name: Build the docs
       run: make docs
   release:
+    permissions:
+      contents: write  # for relekang/python-semantic-release to push release notes and new commits/tags
```

Comment:
I wonder if this one will also trigger the publish event?

Reply:
I have tested this permission and it is enough to publish the package.

Reply:
We’ll find out!